src/TerritorialCouncil/Filter/MembersListFilter.php

```diff
@@ -0,0 +1,150 @@
+<?php
+
+namespace App\TerritorialCouncil\Filter;
+
+use App\Entity\ReferentTag;
+use Symfony\Component\Validator\Constraints as Assert;
+
+class MembersListFilter
+{
+    /**
+     * @var ReferentTag[]
+     *
+     * @Assert\NotNull
+     */
+    private $referentTags = [];
+
+    /**
+     * @var bool|null
+     */
+    private $emailSubscription;
+
+    /**
+     * @var string|null
+     *
+     * @Assert\Length(max=255)
+     */
+    private $firstName;
+
+    /**
+     * @var string|null
+     *
+     * @Assert\Length(max=255)
+     */
+    private $lastName;
+
+    /**
+     * @var string|null
+     */
+    private $gender;
+
+    /**
+     * @var string
+     *
+     * @Assert\NotBlank
+     * @Assert\Choice(choices={"lastName"})
+     */
+    private $sort = 'adherent.lastName';
+
+    /**
+     * @var string
+     *
+     * @Assert\NotBlank
+     * @Assert\Choice(choices={"a", "d"})
+     */
+    private $order = 'a';
+
+    public function __construct(array $referentTags, string $subscriptionType)
+    {
+        $this->referentTags = $referentTags;
+        $this->subscriptionType = $subscriptionType;
```

Comment:
unused property?

Reply:
It's used here https://github.com/EnMarche/en-marche.fr/pull/4749/files#diff-c20f802f969a22bcec26c4f2b7616d0eR72